docs/reference/api/widgets/webview.rst

```diff
@@ -61,20 +61,33 @@ Usage
     # Load static HTML content into the wevbiew.
     webview.set_content("https://example.com", "<html>...</html>")
 
-Notes
------
 
-* Due to app security restrictions, WebView can only display ``http://`` and
-  ``https://`` URLs, not ``file://`` URLs. To serve local file content, run a
-  web server on ``localhost`` using a background thread.
+.. _webview-system-requires:
+
+System requirements
+-------------------
 
 * Using WebView on Windows 10 requires that your users have installed the `Edge
   WebView2 Evergreen Runtime
   <https://developer.microsoft.com/en-us/microsoft-edge/webview2/#download>`__.
   This is installed by default on Windows 11.
 
 * Using WebView on Linux requires that the user has installed the system packages
-  for WebKit2, plus the GObject Introspection bindings for WebKit2.
+  for WebKit2, plus the GObject Introspection bindings for WebKit2. The name of
+  the system package required is distribution dependent:
+
+  - Ubuntu 18.04, 20.04; Debian 10: ``gir1.2-webkit2-4.0``
+  - Ubuntu 22.04+; Debian 11+: ``gir1.2-webkit2-4.1``
+  - Fedora: ``webkit2gtk3``
```

Comment:
So, `webkit2gtk3` will install 4.0 on modern Fedora (and any that aren't EOL). If we're considering this just to encapsulate Fedora distros, then we should probably use `webkit2gtk4.1` instead.

If, however, Fedora is more an umbrella for RHEL, then we may need to break this out like we're doing for Debian...

```suggestion
  - Fedora: ``webkit2gtk4.1``
```

Reply:
I'm happy to keep this Fedora specific - that will be by far the more common usage on desktop. Plus, if a paid RHEL user wants instructions, they can contact their vendor for assistance 🤪